packages/static-hosting/lib/static-hosting.ts

```diff
@@ -139,6 +139,13 @@ export interface StaticHostingProps {
    */
   enableStaticFileRemap?: boolean;
 
+  /**
+   * Any prefixes to remapping that should be included in the path such as au or nz
+   *
+   * @default true
+   */
+  behaviourPrefixes?: { prefix: string; edgeLambdas: EdgeLambda[] }[];
```

Comment:
Not sure what the point of this is :thinking: We already have the `additionalBehaviors` prop and `staticFileRemapPrefixes` to handle prefixes. 

The name might just need updating to make it clear what it does

Reply:
updated this to what we discussed defaultBehaviourPrefixes